recipes/fslpy/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "fslpy" %}
+{% set version = "1.10.1" %}
+{% set sha256 = "d8af3c12a4cac263ad5cb75c84004606faaa99ec5d3c6bfd2f9d3e337726c785" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
Could you please change this part to
```
script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Reply:
@marcelotrevisani  All builds now failing with `Undefined Jinja2 variables remain (['PYTHON'])`. Any ideas?

Reply:
Any problem with me reverting this change? It was working fine before.